api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
IMO the latter. Basically we make it clear in docs that we no longer going to jump listeners. Once you hit a listener, you stay there. IMO much simpler.